src/main/java/conversage/task/Deadline.java

```diff
@@ -0,0 +1,44 @@
+package conversage.task;
+
+import conversage.exception.ConverSageException;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a task with a deadline.
+ */
+public class Deadline extends Task{
+    protected LocalDateTime deadline;
+    protected static final DateTimeFormatter INPUT_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+    protected static final DateTimeFormatter OUTPUT_FORMAT = DateTimeFormatter.ofPattern("MMM d yyyy, h:mma");
```

Comment:
I like that the order of the method modifiers follows the coding standards, as per https://se-education.org/guides/conventions/java/index.html#methods